<?xml version="1.0" encoding="utf-8" standalone="no"?>
<test-run id="2" name="mock-nunit-assembly.dll" fullname="/home/charlie/Dev/NUnit/nunit-3.0/bin/Debug/net-2.0/mock-nunit-assembly.dll" testcasecount="31" result="Failed" total="28" passed="18" failed="5" inconclusive="1" skipped="4" asserts="2" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.147552">
  <environment nunit-version="3.0.5410.34202" clr-version="2.0.50727.1433" os-version="Unix 3.13.0.36" platform="Unix" cwd="/home/charlie/Dev/NUnit/nunit-3.0/bin/Debug" machine-name="cedar" user="charlie" user-domain="cedar" culture="en-US" uiculture="en-US" />
  <test-suite type="Assembly" id="1046" name="mock-nunit-assembly.dll" fullname="/home/charlie/Dev/NUnit/nunit-3.0/bin/Debug/net-2.0/mock-nunit-assembly.dll" runstate="Runnable" testcasecount="31" result="Failed" site="Child" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.082000" total="28" passed="18" failed="5" inconclusive="1" skipped="4" asserts="2">
    <properties>
      <property name="_PID" value="15236" />
      <property name="_APPDOMAIN" value="test-domain-mock-nunit-assembly.dll" />
    </properties>
    <failure>
      <message>One or more child tests had errors</message>
    </failure>
    <test-suite type="TestSuite" id="1047" name="NUnit" fullname="NUnit" runstate="Runnable" testcasecount="31" result="Failed" site="Child" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.074000" total="28" passed="18" failed="5" inconclusive="1" skipped="4" asserts="2">
      <failure>
        <message>One or more child tests had errors</message>
      </failure>
      <test-suite type="TestSuite" id="1048" name="Tests" fullname="NUnit.Tests" runstate="Runnable" testcasecount="31" result="Failed" site="Child" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.074000" total="28" passed="18" failed="5" inconclusive="1" skipped="4" asserts="2">
        <failure>
          <message>One or more child tests had errors</message>
        </failure>
        <test-suite type="TestSuite" id="1049" name="Assemblies" fullname="NUnit.Tests.Assemblies" runstate="Runnable" testcasecount="11" result="Failed" site="Child" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.057000" total="10" passed="4" failed="4" inconclusive="1" skipped="1" asserts="0">
          <failure>
            <message>One or more child tests had errors</message>
          </failure>
          <test-suite type="TestFixture" id="1000" name="MockTestFixture" fullname="NUnit.Tests.Assemblies.MockTestFixture" runstate="Runnable" testcasecount="11" result="Failed" site="Child" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.054000" total="10" passed="4" failed="4" inconclusive="1" skipped="1" asserts="0">
            <properties>
              <property name="Category" value="FixtureCategory" />
              <property name="Description" value="Fake Test Fixture" />
            </properties>
            <failure>
              <message>One or more child tests had errors</message>
            </failure>
            <test-case id="1005" name="FailingTest" fullname="NUnit.Tests.Assemblies.MockTestFixture.FailingTest" runstate="Runnable" seed="1577084752" result="Failed" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.037000" asserts="0">
              <failure>
                <message>Intentional failure</message>
                <stack-trace>at NUnit.Tests.Assemblies.MockTestFixture.FailingTest () [0x00001] in /home/charlie/Dev/NUnit/nunit-3.0/src/NUnitFramework/mock-assembly/MockAssembly.cs:116
</stack-trace>
              </failure>
            </test-case>
            <test-case id="1010" name="InconclusiveTest" fullname="NUnit.Tests.Assemblies.MockTestFixture.InconclusiveTest" runstate="Runnable" seed="613849289" result="Inconclusive" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.001000" asserts="0">
              <reason>
                <message>No valid data</message>
              </reason>
            </test-case>
            <test-case id="1001" name="MockTest1" fullname="NUnit.Tests.Assemblies.MockTestFixture.MockTest1" runstate="Runnable" seed="133455225" result="Passed" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.000000" asserts="0">
              <properties>
                <property name="Description" value="Mock Test #1" />
              </properties>
            </test-case>
            <test-case id="1002" name="MockTest2" fullname="NUnit.Tests.Assemblies.MockTestFixture.MockTest2" runstate="Runnable" seed="802459947" result="Passed" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.000000" asserts="0">
              <properties>
                <property name="Description" value="This is a really, really, really, really, really, really, really, really, really, really, really, really, really, really, really, really, really, really, really, really, really, really, really, really, really long description" />
                <property name="Severity" value="Critical" />
                <property name="Category" value="MockCategory" />
              </properties>
            </test-case>
            <test-case id="1003" name="MockTest3" fullname="NUnit.Tests.Assemblies.MockTestFixture.MockTest3" runstate="Runnable" seed="1658138704" result="Passed" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.000000" asserts="0">
              <properties>
                <property name="Category" value="AnotherCategory" />
                <property name="Category" value="MockCategory" />
              </properties>
              <reason>
                <message>Succeeded!</message>
              </reason>
            </test-case>
            <test-case id="1007" name="MockTest4" fullname="NUnit.Tests.Assemblies.MockTestFixture.MockTest4" runstate="Ignored" seed="1218312642" result="Skipped" label="Ignored" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.001000" asserts="0">
              <properties>
                <property name="Category" value="Foo" />
                <property name="_SKIPREASON" value="ignoring this test method for now" />
              </properties>
              <reason>
                <message>ignoring this test method for now</message>
              </reason>
            </test-case>
            <test-case id="1004" name="MockTest5" fullname="NUnit.Tests.Assemblies.MockTestFixture.MockTest5" runstate="NotRunnable" seed="1168218830" result="Failed" label="Invalid" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.000000" asserts="0">
              <properties>
                <property name="_SKIPREASON" value="Method is not public" />
              </properties>
              <failure>
                <message>Method is not public</message>
              </failure>
            </test-case>
            <test-case id="1009" name="NotRunnableTest" fullname="NUnit.Tests.Assemblies.MockTestFixture.NotRunnableTest" runstate="NotRunnable" seed="1959249430" result="Failed" label="Invalid" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.000000" asserts="0">
              <properties>
                <property name="_SKIPREASON" value="No arguments were provided" />
              </properties>
              <failure>
                <message>No arguments were provided</message>
              </failure>
            </test-case>
            <test-case id="1011" name="TestWithException" fullname="NUnit.Tests.Assemblies.MockTestFixture.TestWithException" runstate="Runnable" seed="34955911" result="Failed" label="Error" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.003000" asserts="0">
              <failure>
                <message>System.Exception : Intentional Exception</message>
                <stack-trace>  at NUnit.Tests.Assemblies.MockTestFixture.TestWithException () [0x00000] in /home/charlie/Dev/NUnit/nunit-3.0/src/NUnitFramework/mock-assembly/MockAssembly.cs:147 
  at (wrapper managed-to-native) System.Reflection.MonoMethod:InternalInvoke (System.Reflection.MonoMethod,object,object[],System.Exception&amp;)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00000] in &lt;filename unknown&gt;:0 </stack-trace>
              </failure>
            </test-case>
            <test-case id="1006" name="TestWithManyProperties" fullname="NUnit.Tests.Assemblies.MockTestFixture.TestWithManyProperties" runstate="Runnable" seed="732490144" result="Passed" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.000000" asserts="0">
              <properties>
                <property name="Size" value="5" />
                <property name="TargetMethod" value="SomeClassName" />
              </properties>
            </test-case>
          </test-suite>
        </test-suite>
        <test-suite type="TestFixture" id="1023" name="BadFixture" fullname="NUnit.Tests.BadFixture" runstate="NotRunnable" testcasecount="1" result="Failed" label="Invalid" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.001000" total="1" passed="0" failed="1" inconclusive="0" skipped="0" asserts="0">
          <properties>
            <property name="_SKIPREASON" value="No suitable constructor was found" />
          </properties>
          <failure>
            <message>No suitable constructor was found</message>
          </failure>
          <test-case id="1024" name="SomeTest" fullname="NUnit.Tests.BadFixture.SomeTest" runstate="Runnable" seed="1997431163" result="Failed" site="Parent" start-time="0001-01-01 00:00:00Z" end-time="0001-01-01 00:00:00Z" duration="0.000000" asserts="0">
            <failure>
              <message>OneTimeSetUp: No suitable constructor was found</message>
            </failure>
          </test-case>
        </test-suite>
        <test-suite type="TestFixture" id="1025" name="FixtureWithTestCases" fullname="NUnit.Tests.FixtureWithTestCases" runstate="Runnable" testcasecount="4" result="Passed" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.012000" total="4" passed="4" failed="0" inconclusive="0" skipped="0" asserts="2">
          <test-suite type="GenericMethod" id="1031" name="GenericMethod" fullname="NUnit.Tests.FixtureWithTestCases.GenericMethod" runstate="Runnable" testcasecount="2" result="Passed" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.001000" total="2" passed="2" failed="0" inconclusive="0" skipped="0" asserts="0">
            <test-case id="1029" name="GenericMethod&lt;Double&gt;(9.2d,11.7d)" fullname="NUnit.Tests.FixtureWithTestCases.GenericMethod&lt;Double&gt;(9.2d,11.7d)" runstate="Runnable" seed="540536299" result="Passed" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.000000" asserts="0" />
            <test-case id="1030" name="GenericMethod&lt;Int32&gt;(2,4)" fullname="NUnit.Tests.FixtureWithTestCases.GenericMethod&lt;Int32&gt;(2,4)" runstate="Runnable" seed="379940153" result="Passed" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.000000" asserts="0" />
          </test-suite>
          <test-suite type="ParameterizedMethod" id="1028" name="MethodWithParameters" fullname="NUnit.Tests.FixtureWithTestCases.MethodWithParameters" runstate="Runnable" testcasecount="2" result="Passed" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.011000" total="2" passed="2" failed="0" inconclusive="0" skipped="0" asserts="2">
            <test-case id="1026" name="MethodWithParameters(9,11)" fullname="NUnit.Tests.FixtureWithTestCases.MethodWithParameters(9,11)" runstate="Runnable" seed="1684488508" result="Passed" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.011000" asserts="1" />
            <test-case id="1027" name="MethodWithParameters(2,2)" fullname="NUnit.Tests.FixtureWithTestCases.MethodWithParameters(2,2)" runstate="Runnable" seed="799071986" result="Passed" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.000000" asserts="1" />
          </test-suite>
        </test-suite>
        <test-suite type="GenericFixture" id="1039" name="GenericFixture&lt;T&gt;" fullname="NUnit.Tests.GenericFixture&lt;T&gt;" runstate="Runnable" testcasecount="4" result="Passed" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.071000" total="4" passed="4" failed="0" inconclusive="0" skipped="0" asserts="0">
          <test-suite type="TestFixture" id="1040" name="GenericFixture&lt;Double&gt;(11.5d)" fullname="NUnit.Tests.GenericFixture&lt;Double&gt;(11.5d)" runstate="Runnable" testcasecount="2" result="Passed" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.001000" total="2" passed="2" failed="0" inconclusive="0" skipped="0" asserts="0">
            <test-case id="1041" name="Test1" fullname="NUnit.Tests.GenericFixture&lt;Double&gt;(11.5d).Test1" runstate="Runnable" seed="181561213" result="Passed" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.000000" asserts="0" />
            <test-case id="1042" name="Test2" fullname="NUnit.Tests.GenericFixture&lt;Double&gt;(11.5d).Test2" runstate="Runnable" seed="830068503" result="Passed" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.000000" asserts="0" />
          </test-suite>
          <test-suite type="TestFixture" id="1043" name="GenericFixture&lt;Int32&gt;(5)" fullname="NUnit.Tests.GenericFixture&lt;Int32&gt;(5)" runstate="Runnable" testcasecount="2" result="Passed" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.002000" total="2" passed="2" failed="0" inconclusive="0" skipped="0" asserts="0">
            <test-case id="1044" name="Test1" fullname="NUnit.Tests.GenericFixture&lt;Int32&gt;(5).Test1" runstate="Runnable" seed="1223302585" result="Passed" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.000000" asserts="0" />
            <test-case id="1045" name="Test2" fullname="NUnit.Tests.GenericFixture&lt;Int32&gt;(5).Test2" runstate="Runnable" seed="552955908" result="Passed" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.000000" asserts="0" />
          </test-suite>
        </test-suite>
        <test-suite type="TestFixture" id="1016" name="IgnoredFixture" fullname="NUnit.Tests.IgnoredFixture" runstate="Ignored" testcasecount="3" result="Skipped" label="Ignored" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.000000" total="3" passed="0" failed="0" inconclusive="0" skipped="3" asserts="0">
          <properties>
            <property name="_SKIPREASON" value="BECAUSE" />
          </properties>
          <reason>
            <message>BECAUSE</message>
          </reason>
          <test-case id="1017" name="Test1" fullname="NUnit.Tests.IgnoredFixture.Test1" runstate="Runnable" seed="153094556" result="Skipped" label="Ignored" site="Parent" start-time="0001-01-01 00:00:00Z" end-time="0001-01-01 00:00:00Z" duration="0.000000" asserts="0">
            <reason>
              <message>OneTimeSetUp: BECAUSE</message>
            </reason>
          </test-case>
          <test-case id="1018" name="Test2" fullname="NUnit.Tests.IgnoredFixture.Test2" runstate="Runnable" seed="1791574301" result="Skipped" label="Ignored" site="Parent" start-time="0001-01-01 00:00:00Z" end-time="0001-01-01 00:00:00Z" duration="0.000000" asserts="0">
            <reason>
              <message>OneTimeSetUp: BECAUSE</message>
            </reason>
          </test-case>
          <test-case id="1019" name="Test3" fullname="NUnit.Tests.IgnoredFixture.Test3" runstate="Runnable" seed="1313028116" result="Skipped" label="Ignored" site="Parent" start-time="0001-01-01 00:00:00Z" end-time="0001-01-01 00:00:00Z" duration="0.000000" asserts="0">
            <reason>
              <message>OneTimeSetUp: BECAUSE</message>
            </reason>
          </test-case>
        </test-suite>
        <test-suite type="ParameterizedFixture" id="1032" name="ParameterizedFixture" fullname="NUnit.Tests.ParameterizedFixture" runstate="Runnable" testcasecount="4" result="Passed" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.070000" total="4" passed="4" failed="0" inconclusive="0" skipped="0" asserts="0">
          <test-suite type="TestFixture" id="1033" name="ParameterizedFixture(42)" fullname="NUnit.Tests.ParameterizedFixture(42)" runstate="Runnable" testcasecount="2" result="Passed" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.000000" total="2" passed="2" failed="0" inconclusive="0" skipped="0" asserts="0">
            <test-case id="1034" name="Test1" fullname="NUnit.Tests.ParameterizedFixture(42).Test1" runstate="Runnable" seed="1746309700" result="Passed" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.000000" asserts="0" />
            <test-case id="1035" name="Test2" fullname="NUnit.Tests.ParameterizedFixture(42).Test2" runstate="Runnable" seed="751833151" result="Passed" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.000000" asserts="0" />
          </test-suite>
          <test-suite type="TestFixture" id="1036" name="ParameterizedFixture(5)" fullname="NUnit.Tests.ParameterizedFixture(5)" runstate="Runnable" testcasecount="2" result="Passed" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.000000" total="2" passed="2" failed="0" inconclusive="0" skipped="0" asserts="0">
            <test-case id="1037" name="Test1" fullname="NUnit.Tests.ParameterizedFixture(5).Test1" runstate="Runnable" seed="1566720606" result="Passed" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.000000" asserts="0" />
            <test-case id="1038" name="Test2" fullname="NUnit.Tests.ParameterizedFixture(5).Test2" runstate="Runnable" seed="748873569" result="Passed" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.000000" asserts="0" />
          </test-suite>
        </test-suite>
        <test-suite type="TestSuite" id="1050" name="Singletons" fullname="NUnit.Tests.Singletons" runstate="Runnable" testcasecount="1" result="Passed" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.071000" total="1" passed="1" failed="0" inconclusive="0" skipped="0" asserts="0">
          <test-suite type="TestFixture" id="1012" name="OneTestCase" fullname="NUnit.Tests.Singletons.OneTestCase" runstate="Runnable" testcasecount="1" result="Passed" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.001000" total="1" passed="1" failed="0" inconclusive="0" skipped="0" asserts="0">
            <test-case id="1013" name="TestCase" fullname="NUnit.Tests.Singletons.OneTestCase.TestCase" runstate="Runnable" seed="573025224" result="Passed" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.000000" asserts="0" />
          </test-suite>
        </test-suite>
        <test-suite type="TestSuite" id="1051" name="TestAssembly" fullname="NUnit.Tests.TestAssembly" runstate="Runnable" testcasecount="1" result="Passed" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.071000" total="1" passed="1" failed="0" inconclusive="0" skipped="0" asserts="0">
          <test-suite type="TestFixture" id="1014" name="MockTestFixture" fullname="NUnit.Tests.TestAssembly.MockTestFixture" runstate="Runnable" testcasecount="1" result="Passed" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.000000" total="1" passed="1" failed="0" inconclusive="0" skipped="0" asserts="0">
            <test-case id="1015" name="MyTest" fullname="NUnit.Tests.TestAssembly.MockTestFixture.MyTest" runstate="Runnable" seed="1954453960" result="Passed" start-time="2014-11-02 21:22:56Z" end-time="2014-11-02 21:22:56Z" duration="0.000000" asserts="0" />
          </test-suite>
        </test-suite>
      </test-suite>
    </test-suite>
  </test-suite>
</test-run>